api/v1/runtime_types.go

```diff
@@ -194,10 +194,21 @@ type Filter struct {
 	Egress  Egress   `json:"egress"`
 }
 
+// Ingress filtering can be enabled for `shoot-networking-fitler` extension with
+// the blackholing feature, see https://github.com/gardener/gardener-extension-shoot-networking-filter/blob/master/docs/usage/shoot-networking-filter.md#ingress-filtering
 type Ingress struct {
+	// It means that the blackholing filtering is enabled.
+	// Either on the per shoot level, or the per worker group level.
 	Enabled bool `json:"enabled"`
+
+	// List of the worker pool names for which the blackholing filtering is enabled on the per worker group level.
+	WorkerNames []*string `json:"workerNames,omitempty"`
```

Comment:
I think both lists of worker names and StaticIPs should be removed from RuntimeCR type